bob/src/cluster/quorum.rs

```diff
@@ -201,11 +203,17 @@ impl Quorum {
             return Err(Error::internal());
         }
         trace!("selection of free nodes available for data writing");
-        let sup_nodes = self.mapper.get_support_nodes(key, failed_nodes.len());
-        debug!("PUT[{}] sup put nodes: {:?}", key, &sup_nodes);
-        let nodes_need_remote_backup: Vec<_> = failed_nodes.drain(..sup_nodes.len()).collect();
-        let queries: Vec<_> = sup_nodes
+        let primary_sup_nodes = self.mapper.get_support_nodes(key, failed_nodes.len());
+        let secondary_sup_nodes = self.get_secondary_support_nodes(&primary_sup_nodes);
```

Comment:
`mapper` already returns as match support nodes as possible. We don't need any `secondary` nodes here. If `mapper.get_support_nodes` returns less nodes than required, then the rest should be written to local node

Reply:
So basically just revert all changes in `put_aliens` function